code/game/objects/items/weapons/grenades/syndieminibomb.dm

```diff
@@ -22,7 +22,7 @@
 	if(!active)
 		flags |= NODROP
 		to_chat(user, "<span class='userdanger'>As you activate the bomb, it emits a substance that sticks to your hand! It won't come off!</span>")
-		to_chat(user, "<span class='sans'>Uh oh.</span>")
```

Comment:
personal opinion but its funnier without the ellipses. Its also a reference to the `Comic` disability and comical implant.